modules/core/types/codec.go

```diff
@@ -7,16 +7,12 @@ import (
 	connectiontypes "github.com/cosmos/ibc-go/v6/modules/core/03-connection/types"
 	channeltypes "github.com/cosmos/ibc-go/v6/modules/core/04-channel/types"
 	commitmenttypes "github.com/cosmos/ibc-go/v6/modules/core/23-commitment/types"
-	solomachinetypes "github.com/cosmos/ibc-go/v6/modules/light-clients/06-solomachine"
-	ibctm "github.com/cosmos/ibc-go/v6/modules/light-clients/07-tendermint"
 )
 
```

Comment:
What do you think about adding an alias to the func signature for context? 
Not sure about naming, if it makes sense etc...

```suggestion

// ClientRegistrar handles registration of light-client interface types
type ClientRegistrar func(registry codectypes.InterfaceRegistry)

```

Reply:
I was thinking of doing something like this, technically this functionality isn't limited in scope to registration functions for clients, it would let you register any interface types. That's why I used `additionalRegistration` rather than `clientRegistration`.  We could do something like `type InterfaceRegistrar func(registry codectypes.InterfaceRegistry)`?

Reply:
Aye, that's kind of what I was getting at with using an alias for the func signature here. I know you could technically pass anything, but it would be nice to infer that the intension is to provide register interfaces funcs for client modules.

I like `type InterfaceRegistrar func(registry codectypes.InterfaceRegistry)`, but we could then name the variable something more client specific.

Reply:
I'm happy with your original suggestion of `ClientRegistar` @damiannolan , I've implemented that change, we can always rename if we like a different name better!